### ISSUE TITLE
meteor-kitchen install script - cleanup mistake

### ISSUE THREAD

**Issue by mikkelking** [perak/kitchen-site#128]
Petar,

Part of the output of the installation:
```
Unpacking...
Cleaning up...
rm: /Users/triton/.meteor-kitchen/examples/example-*.json: No such file or directory
Done.
```
I think the rm statement should be
```
rm /Users/triton/.meteor-kitchen/examples/example-*/*.json
```

**Comment by perak** [comment]
@mikkelking that `rm` command in install script is outdated. 

Before, all examples was in single directory `/examples/`, when I moved examples to separate directories, I added this command to remove old examples.

I just removed that command from install script (it will be available in next version 0.9.49)

Thanks!

**Comment by perak** [comment]
@mikkelking solved (long time ago :) ). Closing issue.
